docs/src/conf.py

```diff
@@ -92,12 +93,16 @@ def autolog(message):
 
 # -- General configuration ---------------------------------------------------
 
-# Create a variable that can be insterted in the rst "|copyright_years|".
-# You can add more vairables here if needed
+# Create a variable that can be inserted in the rst "|copyright_years|".
+# You can add more variables here if needed.
 rst_epilog = """
```

Comment:
Is there a reason for using format strings in the `.format` method, rather than having a single format string 
```
f"""
|copyright_years| replace:: 2010 - {upper_copy_year}
[...etc]
"""
```

Reply:
Yup, could do, thanks.

I was lazily following what was there before... I'll tidy it up :+1:

Reply:
Done. Nice, that's way tidier, thanks.